src/main/java/core/basesyntax/FigureBehaviour.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface FigureBehaviour {
+    void draw();
+
+    int area();
```

Comment:
<img width="1002" alt="Screenshot 2022-01-20 at 10 09 28" src="https://user-images.githubusercontent.com/33026471/150298511-87ae5aeb-5396-45b5-b927-0cb5530a9add.png">
let's create FigureDrawer and AreaCalculator interfaces